edge-agent-sdk/src/commonMain/kotlin/org/hyperledger/identus/walletsdk/edgeagent/EdgeAgent.kt

```diff
@@ -991,6 +999,16 @@ class EdgeAgent {
                 }
             }
 
+            val linkSecret = getLinkSecret()
+
+            val presentation = pollux.createVerifiablePresentationAnoncred(
+                request,
+                credential as AnonCredential,
+                linkSecret
+            )
```

Comment:
How is this coded in Pollux interface side? Is `AnonCredential` defined in the domain?

Reply:
Good catch, that code was duplicated. I cleaned that up.

Reply:
;)